src/main/java/bhs/devilbotz/commands/DriveCommand.java

```diff
@@ -53,17 +54,32 @@ public void initialize() {}
    */
   @Override
   public void execute() {
+    // Clamp the values to the range [0.95, 0.95] to allow the PID loop some room to work
+    // TODO: replace clamp with dynamic range
+    double speed = MathUtil.clamp(this.speed.getAsDouble(), -0.95, 0.95);
+    double rot = MathUtil.clamp(this.rot.getAsDouble(), -0.95, 0.95);
+
+    // Add a deadband to the joystick values
+    speed = MathUtil.applyDeadband(speed, Constants.DriveConstants.JOYSTICK_DEADBAND);
+    rot = MathUtil.applyDeadband(rot, Constants.DriveConstants.JOYSTICK_DEADBAND);
```

Comment:
Shouldn't the deadband be added to the joystick output?

Reply:
It is, speed and rot are using the raw joystick values. The dead band is applied before the curve and other methods.

Reply:
Ah, gotcha. The variables speed and rot were confusing.  I was expecting something like X/Y, but since this is arcade drive, this makes sense.  Tnx.